HowToFIDO.md

```diff
@@ -106,7 +106,23 @@ utilize the APIs correctly, e.g., by requesting the appropriate
 transport or attachment type for the particular use case they are
 addressing. If not done correctly, you risk confusing your users, making
 authentication more complicated than necessary for them, or locking them
-out of their account. Below is an overview of the use cases unlocked by
+out of their account. 
+
+### 1.1 Platform Authenticators and Roaming Authenticators
+
+Platform authenticators have a unique issue which needs to be carefully considered when designing the flows and operations. 
+
+**Issue:** Platform authenticators cannot be connected to and used with other devices. Namely, they cannot bootstrap other devices [16].
+
```

Comment:
Perhaps start with a short recap of the difference between platform and roaming authenticators? Or if we just want to focus on the issue of lockout perhaps we can change the "Overview" section to "FIDO2 Usecases", and then add a 1.1 "Special considerations for Platform Authenticators"? In the future perhaps we can add 1.2 "Special considerations for Roaming Authenticators"?